template/sbt.go

```diff
@@ -13,7 +13,7 @@ sbtscratch:
     {{.ScratchVolumes}}
 {{end}}
 sbt: &sbt
-  image: arch-docker.eng.lancope.local:5000/sbt
+  image: test.local:5000/sbt
```

Comment:
Why don't we publish an sbt image to Docker Hub and make that the default. That way it would at least work if they don't override it.